Changes

```diff
@@ -1,3 +1,12 @@
+0.54.x.x (relative to 0.54.2.1)
```

Comment:
0.54.2.x (relative to 0.54.2.2)

Reply:
Ahh yes, sorry, of course, no new features ;) Fixed in https://github.com/GafferHQ/gaffer/pull/3474/commits/0301b8544cf37a250b00a7d79f88d89c0431078f.